project-base/storefront/hooks/useBroadcastChannel.ts

```diff
@@ -19,7 +26,9 @@ export const useBroadcastChannel = (name: BroadcastChannelsType, callBack: (data
         }
 
         channel.onmessage = (event) => {
-            callBack(event.data);
+            if (event.data.tabId !== tabId && broadcastChannelSameTabConfig[name]) {
```

Comment:
Have you left `event.data.tabId !== tabId` intentionally? This disable calling callback in current tab which invoke this broadcast message. According to our discussion this might be wanted, right?

Reply:
I had, but I did not change the logical operator. I rewrote it to
`if (event.data.tabId !== tabId || broadcastChannelSameTabConfig[name]) {`
so that it is either in different tab (always OK, we dont care), or if it is in the same tab, then it must be allowed.